cmd/billing.go

```diff
@@ -54,17 +54,23 @@ func init() {
 	cmds = append(cmds, billingCmd)
 }
 
+// Error returned when attempting to use a team token to perform user actions
+var errUserActionAsTeam = cli.NewExitError("Cannot perform user action as team", -1)
+
 func addBillingProfileCmd(cliCtx *cli.Context) error {
 	ctx := context.Background()
 
-	userID, err := loadUserID(ctx)
-	if err != nil {
-		return err
+	userID, userIDErr := loadUserID(ctx)
+	if userIDErr != nil {
+		return userIDErr
 	}
 
-	teamID, err := validateTeamID(cliCtx)
-	if err != nil {
-		return err
+	teamID, teamIDErr := validateTeamID(cliCtx)
+	if teamIDErr != nil {
+		return teamIDErr
+	}
+	if teamID == nil && userIDErr == errUserActionAsTeam {
```

Comment:
look at line 64, if userIDErr is != nil this function returns, so it can never be equal to errUSerActionAsTeam

Reply:
Added a test above to ensure we don't return early when it's `errUserActionAsTeam`

Reply:
:+1: